spec/support/FailingServer.js

```diff
@@ -6,14 +6,20 @@ const ParseServer = require('../../lib/index').ParseServer;
 
 const databaseURI = 'mongodb://doesnotexist:27017/parseServerMongoAdapterTestDatabase';
 
-ParseServer.start({
-  appId: 'test',
-  masterKey: 'test',
-  databaseAdapter: new MongoStorageAdapter({
-    uri: databaseURI,
-    mongoOptions: {
-      serverSelectionTimeoutMS: 2000,
-    },
-  }),
-  filesAdapter: new GridFSBucketAdapter(databaseURI),
-});
+(async () => {
+  try {
+    await ParseServer.startApp({
+      appId: 'test',
+      masterKey: 'test',
+      databaseAdapter: new MongoStorageAdapter({
+        uri: databaseURI,
+        mongoOptions: {
+          serverSelectionTimeoutMS: 2000,
+        },
+      }),
+      filesAdapter: new GridFSBucketAdapter(databaseURI),
+    });
+  } catch (e) {
+    process.exit(1);
+  }
+})();
```

Comment:
Could you also refactor this to remove the IIFE?

Reply:
`SyntaxError: await is only valid in async functions and the top level bodies of modules`

Reply:
Got it, it's only a test, so I guess we can keep it as is